db/schema.rb

```diff
@@ -145,7 +145,7 @@
   end
 
   create_table "projects", force: :cascade do |t|
-    t.bigint "user_id", null: false
+    t.bigint "user_id"
```

Comment:
does it make sense for projects to have a history of users who worked on it?

Reply:
it does makes sense for a project to have an actively assigned single finisher

Reply:
nm this is for the owner of the project.  what i'm asking about is captured in assignments